docs/docs/administration/install-from-scratch.md

```diff
@@ -273,14 +273,14 @@ If you prefer nginx or another proxy, please refer to their documentation.
 !!! info "Optional"
     You can skip this step if you only plan to use Mathesar from `localhost`.
 
-1. Ensure that your DNS `A` and/or `AAAA` records are configured correctly. `<your-domain>` should resolve to your server's IP.
+1. Ensure that your DNS `A` and/or `AAAA` records are configured correctly. Your domain should resolve to your server's IP.
 
 1. Enter your domain and press <kbd>Enter</kbd> to customize the remaining steps in this guide.
 
     <input data-input-for="DOMAIN_NAME" aria-label="Your Domain name "/>
 
     - For example: `example.com`
-    - Do _not_ precede your domain with `https://`
+    - Do _not_ precede your domain with `http://` or `https://`
```

Comment:
localhost might not work without `http://` with caddy if it tries to get a SSL certificate from let's encrypt, might be a good idea to test this.

I had ran into this issue while writing our prod docker-compose.

Reply:
We might have to specify users to only deploy with a reverse proxy if they're using a domain, or if they know what they're doing.

Reply:
Updated in https://github.com/mathesar-foundation/mathesar/pull/4508/commits/43d08255fe1af6fba29c529bfb08a4251c846dfa